plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -27,28 +27,40 @@ SecRule TX:nextcloud-rule-exclusions-plugin_enabled "@eq 0" "id:9508099,phase:1,
 # you put something like this in crs-setup.conf:
 #
 # SecRule REQUEST_FILENAME "@rx /(?:remote\.php|index\.php)/" \
-#   "id:9508600,\
-#   phase:2,\
-#   t:none,\
-#   nolog,\
-#   pass,\
-#   setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
+#     "id:9508600,\
+#     phase:2,\
+#     t:none,\
+#     nolog,\
+#     pass,\
+#     setvar:'tx.restricted_extensions=.bak/ .config/ .conf/'"
 #
 # Large uploads can be modified with SecRequestBodyLimit. Or they
 # can be more controlled by using the following:
 #
 # SecRule REQUEST_URI "@endsWith /index.php/apps/files/ajax/upload.php" \
-#    "id:9508610,\
-#    phase:1,\
-#    t:none,\
-#    nolog,\
-#    ctl:requestBodyLimit=1073741824"
+#     "id:9508610,\
+#     phase:1,\
+#     t:none,\
+#     nolog,\
+#     ctl:requestBodyLimit=1073741824"
 #
 # Please note, that `ctl:requestBodyLimit` action does not work
 # in libmodsecurity3!
+#
+# The Nextcloud desktop client occasionally sends large request bodies not containing any uploaded files.
```

Comment:
I think this new section should live in the readme instead. Maybe reference the readme in the paragraph on lines 37, 38 for this particular case.

Reply:
Agreed, I've documented them in readme so they aren't hidden.